metricbeat/mb/module/options.go

```diff
@@ -64,14 +64,14 @@ func WithMetricSetInfo() Option {
 // given to the `service.name` setting in the module configuration.
 func WithServiceName() Option {
 	return func(w *Wrapper) {
+		serviceName := w.Module.Config().ServiceName
```

Comment:
I know it was like this before but `w.Module` is an interface and it might be `nil`

Reply:
You are right, checking it here won't do any harm :slightly_smiling_face: Thanks!